docs/semgrep-app/findings.md

```diff
@@ -95,42 +95,72 @@ The Findings page consists of:
 | **Projects**     | Filter by repositories connected to Semgrep App. |
-| **Branches**     | Filter by findings in different branches. |
-| **Rules**        | Filter by rules or rulesets that are included in your Rule Board. More than one rule can be selected for filtering. |
-| **Rulesets**     | Filter by name of the ruleset where rules that matched the code belong. |
+| **Status**       | Filter for different findings triage states. Refer to the [following table](#triaging-findings) to understand triage states. |
+| **Severities**    | Filter by the severity of a finding. Possible values: <ul><li>Low</li><li>Medium</li><li>High</li></ul> |
 | **Actions**     | Filter by monitoring, commenting, or blocking rules in your Rule Board. |
-| **Severity**    | Filter by the severity of a finding. Possible values: <ul><li>Low</li><li>Medium</li><li>High</li></ul> |
+| **Confidences** | Filter by indication of the rule to detect true positives. The higher the confidence the more true positives the rule may detect.  |
+| **Categories**  |  Filter by various rule categories, such as security, or best practice rules. More than one category can be selected for filtering. |
+| **Branches**     | Filter by findings in different Git branches. More than one branch can be selected for filtering. |
+| **Rules**        | Filter by rules or rulesets that are included in your Rule Board. More than one rule can be selected for filtering. More than one rule can be selected for filtering. |
+| **Rulesets**     | Filter by name of the ruleset where rules that matched the code belong. More than one rule or ruleset can be selected for filtering. |
+
+You can also filter for findings reported from specific time periods or group findings by rule. For more information see 
+
+To filter through findings:
+
+1. Click the filter criteria.
+1. The page then refreshes to reflect the additional criteria.
+1. Optional: To futher refine your filter, select additional filters.
+
+#### Displaying findings reported in specific time
 
-To filter through all findings:
+Display and filter for findings reported only in specific time. To display findings reported in a specific time, follow these steps:
 
-1. Click the filter criteria drop-down box.
-2. Enable displayed checkbox or write in a field value by which the findings are filtered.
-3. The page then refreshes to reflect the additional criteria.
-4. Additional values may be selected to further refine your filter.
+1. In Semgrep App, go to **[Findings](https://semgrep.dev/orgs/-/findings)** page.
+1. Click the <i class="fa-solid fa-calendar-days"></i> **All time** button.
+1. Select preferred time period for which to display findings. The following options are available:
+    - **All time** - This is the default option.
+    - Last 1 year
+    - Last 1 month
+    - Last 7 days
+    - Last 1 day
+
+### Grouping by rule
```

Comment:
This is still in beta. (Needs a feature flag) Not sure if we should include it yet?

Reply:
@annikalee ok...

Reply:
Solutions:
1. Remove this part of the documentation and keep it in another PR.
2. Mark this feature as a beta for specific customers who may request it.

Reply:
@adamkvitek Channeling my inner Raghav here, I’d go for #2 but you may want to confirm with Chinmay.

Reply:
@neo413001 we were thinking with @s-santillan that solution 1. (see the conversation above in this thread) might be better for now. Should we mark this section about grouping by rule as beta or move it to another PR and wait until the feature is generally out for everyone? I think that if we mark it as beta we need to say that users need to contact us to get this beta feature.